nixos/tests/networking.nix

```diff
@@ -463,7 +463,9 @@ let
         };
       };
 
-      testScript = ''
+      # The test is disabled for systemd-networkd.
+      # See the commit message of a3a441c.
+      testScript = optionalString (!networkd) ''
```

Comment:
This sounds like the wrong place to do this. Shouldn't that be done in line 449 and just set to `null` in case of networkd?

Reply:
I tried setting the attrset to `null` or `{ }` but it doesn't work.

Reply:
ping. I'd like to fix this before the final release of 20.03